### DISCUSSION TITLE
TSS enrichment plots

### DISCUSSION THREAD

**Discussion by genomaxx** [stuart-lab/signac#443] [Q&A] [answered]
Hello!

I was curious about the interpretation of a 'fuzzy' TSS enrichment plot? I am quite used to seeing tighter lines traced in this plot; what does it mean when the vertical lines extend a few units? I haven't seen this before either in the 10X cellranger output (for any sample I have run) nor in the vignettes for Signac.

Two examples below.
The first pair of plots have longer vertical strokes than I expected, but not unduly so. The second set however, specifically for the 'high' case, has extremely long strokes and consequently has a bit of a 'fuzzy' appearance.

<img width="867" alt="example1" src="https://user-images.githubusercontent.com/15305325/107594518-eaaf8480-6bc6-11eb-945b-203662f1713c.png">
<img width="866" alt="example2" src="https://user-images.githubusercontent.com/15305325/107594522-ee430b80-6bc6-11eb-96c3-9cf9116b38f6.png">

Thanks!

**Comment by timoast** [comment, marked as answer]
This may be due to sparse data, does one dataset have many more cells than the other, or many more counts per cell? Alternatively, for the second plot, this could happen if the coordinates of the TSSs are wrong (wrong genome build, for example)

**Reply by genomaxx** [reply]
So for the first dataset, there were 1135 cells and a median of 40,245 fragments per cell (30% of frags. in targeted reg.). I believe this is quite deep sequencing. Would a relatively low fraction of fragments in the targeted regions affect peak height/shape as well as the vertical distance covered by each line? The second dataset had 2919 cells and a median of 4735 fragments per cell (70% of frags. in targeted reg.) - still well within the acceptable ranges afaik. 

Re: wrong genome build for dataset 2 - you are absolutely right, I went back and double checked and was using the incorrect build! Thx! If I may ask, what tipped you off to the incorrect TSS coordinates explanation? What calculation determines the height of the lines?

**Reply by timoast** [reply]
For the second plot you do not see the distinctive peak centered at the TSS, followed by the smaller peak ~1 nucleosome downstream. This indicates that either there is extremely poor targeting of Tn5 to open chromatin regions, or the TSS positions are wrong.
